service/src/main/java/bio/terra/cbas/util/BackfillOriginalWorkspaceIds.java

```diff
@@ -0,0 +1,135 @@
+package bio.terra.cbas.util;
+
+import bio.terra.cbas.config.CbasContextConfiguration;
+import bio.terra.cbas.dao.MethodDao;
+import bio.terra.cbas.dao.MethodVersionDao;
+import bio.terra.cbas.dao.RunSetDao;
+import bio.terra.cbas.models.Method;
+import bio.terra.cbas.models.MethodVersion;
+import bio.terra.cbas.models.RunSet;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+
+/*
+NOTE: This class is inherently temporary.  Once all original workspace IDs have been backfilled,
+      this entire class should be deleted, along with its method calls in BackfillOriginalWorkspaceIdBean.java
+ */
+public class BackfillOriginalWorkspaceIds {
```

Comment:
Now we know how.....

after Doug's presentation this morning I'd be tempted to:
- Make this a `BackfillOriginalWorkspaceIdsService`, with an `@Service` annotation.
- Have the Dao dependencies autowired instead of provided via parameters per method
- Wrap the chains of interactions with `@WriteTransaction` to make sure there are no race conditions possible

I actually don't think it would make _that_ much difference in this particular case, because we will never be writing un-backfilled values so there's no way for them to slip in unnoticed... but it might be good practice